locale/en/blog/advisory-board/advisory-board-update.md

```diff
@@ -13,7 +13,7 @@ release of v0.12.
 The interim [advisory
 board](https://www.joyent.com/blog/node-js-advisory-board) has met three times
 since its creation. You can find the minutes from the advisory board meetings
-here: [http://nodejs.org/advisory-board](http://nodejs.org/advisory-board). As
+here: [https://nodejs.org/advisory-board](https://nodejs.org/advisory-board). As
```

Comment:
Redirects to https://nodejs.org/en/about/organization/ (as there is no Advisory Board any more)